docs/installing/monsterpi.mdx

```diff
@@ -2,40 +2,95 @@
+[MonsterPi](https://github.com/fdm-monster/MonsterPi) is a [Raspberry Pi](https://www.raspberrypi.org/) image based on [Ubuntu](https://ubuntu.com/download/raspberry-pi) built using [CustomPiOS](https://github.com/guysoft/CustomPiOS) running [FDM Monster](https://fdm-monster.net/).
 
-All released image versions can be downloaded from the [Releases](https://github.com/fdm-monster/MonsterPi/releases)
-page.
+Depending on the image, either [MongoDB](https://www.mongodb.com/) or [SQLite](https://www.sqlite.org/) is included.
 
-## Latest Version
+## Hardware requirements
+
+:::warning
+An Orange Pi or any other Single Board Computer (SBC) has not been tested and will most likely not work.
+:::
+
+We advise you to get these components for your setup. You might already have these or equivalents of them. All links point to the official Raspberry Pi website. From there links to (local) stores are provided.
+
+| Component | Raspberry Pi 3 | Raspberry Pi 4 | Raspberry Pi 5 |
+|---|---|---|---|
+| Raspberry Pi | [Link](https://www.raspberrypi.com/products/raspberry-pi-3-model-b/) | [Link](https://www.raspberrypi.com/products/raspberry-pi-4-model-b/) | [Link](https://www.raspberrypi.com/products/raspberry-pi-5/) |
+| Raspberry Pi Case | [Link](https://www.raspberrypi.com/products/raspberry-pi-3-case/) | [Link](https://www.raspberrypi.com/products/raspberry-pi-4-case/) | [Link](https://www.raspberrypi.com/products/raspberry-pi-5-case/) |
+| Cooling | Not Required | [Link](https://www.raspberrypi.com/products/raspberry-pi-4-case-fan/) | Included with case |
+| Power Supply | [Link](https://www.raspberrypi.com/products/micro-usb-power-supply/) | [Link](https://www.raspberrypi.com/products/type-c-power-supply/) | [Link](https://www.raspberrypi.com/products/27w-power-supply/) |
+| 16GB or bigger Class 10 SD Card<br />Please get a decent brand<br />**do not save money on this**| [Link](https://www.raspberrypi.com/products/sd-cards/) |[Link](https://www.raspberrypi.com/products/sd-cards/)|[Link](https://www.raspberrypi.com/products/sd-cards/)|
+
+Additionally, depending on your device, you might need a SD Card reader and (optionally) an micro SD Card to SD Card adapter.
 
-[![Latest Version](https://img.shields.io/github/v/release/fdm-monster/monsterpi.svg?color=d52828&label=stable)](https://github.com/fdm-monster/monsterpi/releases/latest)
+## Latest Version
 
-The downloaded file needs to be unzipped before flashing it to your SD Card. Use your favorite unzipping application to
-extract the files.
+[![Latest Version](https://img.shields.io/github/v/release/fdm-monster/monsterpi.svg?color=1eb6c3&label=stable)](https://github.com/fdm-monster/monsterpi/releases/latest)
 
 :::note
-Older MonsterPi releases can be downloaded from [GitHub Releases](https://github.com/fdm-monster/MonsterPi/releases).
+Previous MonsterPi images can be downloaded from [GitHub Releases](https://github.com/fdm-monster/MonsterPi/releases).
```

Comment:
```suggestion
The latest and previous MonsterPi images can be downloaded from [GitHub Releases](https://github.com/fdm-monster/MonsterPi/releases).
```

Reply:
I disagree; `latest` is already mentioned above. While *technically* correct, it might add confusion for the end user